gradle.properties

```diff
@@ -19,5 +19,5 @@ ideaVersion = 2017.2
 intellijRepoUrl = https://storage.googleapis.com/cloud-tools-for-java-team-kokoro-build-cache/idea-distributions
 javaVersion = 1.8
 ijPluginRepoChannel = alpha
-version = 17.11.2-SNAPSHOT
+version = 17.12.2-SNAPSHOT
```

Comment:
17.12.1?

Reply:
This represents the next version, not the current one.